src/private_cuda.h

```diff
@@ -46,6 +46,24 @@
 /* Keep in sync with the copy in gpuarray/extension.h */
 #define DONTFREE 0x10000000
 
+#define GA_CUDA_EXIT_ON_ERROR(ctx, cmd) \
+  do {                                  \
+    int err = (cmd);                    \
```

Comment:
Leave it for now.  I will get around to testing on windows and this will be part of the things I'll have to fix.